pkg/cluster/cluster.go

```diff
@@ -79,7 +79,7 @@ func HasClusters(client *cmv1.ClustersClient, creatorARN string) (bool, error) {
 		Size(1).
 		Send()
 	if err != nil {
-		return false, fmt.Errorf("Failed to list clusters: %v", err)
+		return false, fmt.Errorf(response.Error().Reason())
```

Comment:
missing `eporter.Debugf` call?

Reply:
This is part of the clusterprovider, and doesn't have access to the reporter. Reporter errors happen at the `cmd` level (so whichever command calls this code should already have a call to `reporter.Debugf` inside their err-catch